code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -179,9 +179,9 @@
 			if (H.stats.getStat(STAT_ROB) >= 80)
 				var softcap = H.stats.getStat(STAT_ROB) / 10
 				var newrob = (72 + softcap) / 10
-				stat_damage = 3 + max(0, newrob)
+				stat_damage = 3 + max(0, newrob) + H.punch_damage_increase // This makes knuckledusters actually work since NOTHING called this var, ever
 			else
-				stat_damage = 3 + max(0, (H.stats.getStat(STAT_ROB) / 10))
+				stat_damage = 3 + max(0, (H.stats.getStat(STAT_ROB) / 10)) + H.punch_damage_increase
```

Comment:
```suggestion
				stat_damage = 3 + max(0, (H.stats.getStat(STAT_ROB) / 10))
```

Reply:
On a 40 ROB character (Foreman), it went from 8.9 damage with unarmed attacks to a naked human mob to 22 damage with gold knuckle dusters which provide +15 punching strength, (it should've been ~24)
Took off the gloves, punching damage is back to normal
Only the HULK mutation **doubles** punch increases, this is a `+` not a `*` meaning flat damage added